ocpmodels/modules/scaling/compat.py

```diff
@@ -0,0 +1,72 @@
+import json
+import logging
+from pathlib import Path
+from typing import Dict, Optional, Union
+
+import torch
+import torch.nn as nn
+
+from .scale_factor import ScaleFactor
+
+ScaleDict = Union[Dict[str, float], Dict[str, torch.Tensor]]
+
+
+def _load_scale_dict(scale_file: Optional[Union[str, ScaleDict]]):
+    """
+    Loads scale factors from either:
+    - a JSON file mapping scale factor names to scale values
+    - a python dictionary pickled object (loaded using `torch.load`) mapping scale factor names to scale values
+    - a dictionary mapping scale factor names to scale values
+    """
+    if not scale_file:
+        return None
+
+    if isinstance(scale_file, dict):
+        return scale_file
+
+    path = Path(scale_file)
+    if not path.exists():
+        return None
```

Comment:
Log an error message here that the path was passed in the config but doesn't exist, otherwise might silently hide errors.